newdle/client/src/App.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
-import './App.scss';
+import {Header} from 'semantic-ui-react';
+
+import styles from './App.module.scss';
 
 export default function App() {
   return (
-    <div>
-      <header>
-        Hello World!
+    <div className={styles.newdleLayout}>
```

Comment:
oh no, this forces us to use camelCase for css class names :(

Anyway, do we even need a class for this? Can we use HTML5 elements? There are quite a few besides `<header>` so I think there might be a suitable one for this one...

Reply:
`<main>`?